src/common/taskmgr.h

```diff
@@ -64,7 +64,7 @@ class CTaskMgr
         std::any InitData,
         TASKTYPE InitType,
         TaskFunc_t InitFunc,
-        duration InitInterval=1s);
+        duration InitInterval);
```

Comment:
It doesn't compile otherwise. Perhaps I'm using an older MSVC version.

Reply:
The CI boxes can build it (well, other than the warning as error on x64 on appveyor on this branch), so it could be your MSVC version.  We only support 2017 on whatever the latest update is (15.3 or something).

at any rate, could you put the default back?

Reply:
Done. https://github.com/DarkstarProject/darkstar/pull/4729/commits/3851ce978753a54d2977ae8e95806aa41e7e9036